main.go

```diff
@@ -44,6 +46,8 @@ func suicide(err error, critical bool) {
 	if !doNotFail || critical {
 		os.Exit(1)
 	}
+
+	exitCode = 254 // not typical error code, for "git grep" friendliness
```

Comment:
... with `--do-not-fail` in effect `suicide()` will not abort execution, but will set not-zero error code for future `os.Exit()`